samples/server/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	"go.opencensus.io/trace"
+	"gocloud.dev/gcp"
+	"gocloud.dev/server"
+	"gocloud.dev/server/sdserver"
+)
+
+type GlobalMonitoredResource struct {
+	projectId string
+}
+
+func (g GlobalMonitoredResource) MonitoredResource() (string, map[string]string) {
+	return "global", map[string]string{"project_id": g.projectId}
+}
+
+func main() {
+	addr := flag.String("listen", ":8080", "HTTP port to listen on")
+
+	ctx := context.Background()
+	credentials, err := gcp.DefaultCredentials(ctx)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	tokenSource := gcp.CredentialsTokenSource(credentials)
+
+	projectId, err := gcp.DefaultProjectID(credentials)
+	fmt.Printf("projectId = %s\n", projectId)
+	if err != nil {
+		log.Fatal(err)
+	}
+	mr := GlobalMonitoredResource{projectId: string(projectId)}
+	exporter, _, err := sdserver.NewExporter(projectId, tokenSource, mr)
```

Comment:
Even for sample code, it's usually nice to give the user to option to stop traces from being uploaded to Stackdriver (they're hard to remove IIRC and pollute your project's traces).

Reply:
Are you thinking some sort of handler / special path for that? I'm not sure this is critical for an example since we're really trying to demonstrate the most simple way to instantiate a server with tracing, and there's an large spectrum of app-specific logic that could be added on top.

Reply:
I was thinking a flag that switches between using a `nil` exporter and not. It's a pretty common need, so I think it's instructive to include, but up to you.

Reply:
Done